package.nix

```diff
@@ -225,6 +226,10 @@ in {
     libsodium
     openssl
     sqlite
+    (toml11.overrideAttrs (old: {
+      # TODO change in Nixpkgs, Windows works fine.
+      meta.platforms = lib.platforms.all;
```

Comment:
https://github.com/NixOS/nixpkgs/pull/322977

Reply:
Thank you!!